### PR TITLE
major changes: add parameter offsets page

### PR DESCRIPTION
**Pull request by oliver-sanders**
A change that we forgot about which was recently noticed: https://github.com/cylc/cylc-flow/pull/3452

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.

### CODE REVIEW

File: src/7-to-8/major-changes/parameters.rst [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+           b_x1 => c_x1  # x=1
+   a_x1 => b_x2 => c_x2  # x=2
+   a_x2 => b_x3 => c_x3  # x=3
+
+Whereas Cylc 8 will remove the entire line:
+
+.. code-block:: cylc-graph
+
+   a_x1 => b_x2 => c_x2  # x=2
+   a_x2 => b_x2 => c_x2  # x=3
```
Comment (oliver-sanders) [inline src/7-to-8/major-changes/parameters.rst:50]: I'm struggling to remember this one it was so long ago.

Is this correct, or does Cylc remove the part of the line after the first out-of-bounds parameter?

File: src/7-to-8/major-changes/parameters.rst [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+Cylc 7 removed the part of the expression which was out of range resulting in a
+partial evaluation of that line:
+
+.. code-block:: cylc-graph
+
+           b_x1 => c_x1  # x=1
+   a_x1 => b_x2 => c_x2  # x=2
+   a_x2 => b_x3 => c_x3  # x=3
+
+Whereas Cylc 8 will remove the entire line:
```
Comment (oliver-sanders) [inline src/7-to-8/major-changes/parameters.rst:45]: I think this is correct:

```suggestion
Whereas Cylc 8 will remove everything after the first out-of-range parameter - `<x-1>` (so the entire line for this example):
```

Comment (hjoliver) [reply src/7-to-8/major-changes/parameters.rst:45]: Yep.

File: src/7-to-8/major-changes/parameters.rst (hunk 1 above)
Comment (hjoliver) [reply src/7-to-8/major-changes/parameters.rst]: (you got it right in the other comment).

**Review by hjoliver — APPROVED** [review]

**Pull request merged**